tests/execute.rs

```diff
@@ -118,38 +187,130 @@ fn execute_script_generic() {
 }
 
 #[test]
-#[ignore = "to be implemented"]
 /// Test execution of a script with correct parameters which stores something inside the storage.
 fn execute_script_storage_correct() {
-    new_test_ext().execute_with(|| {
-        assert_eq!(1, 0);
-    });
-}
+    pub const ALICE_ADDR: &str = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY";
+
+    let (bob_addr_32, bob_addr_mv) = addrs_from_ss58(BOB_ADDR).unwrap();
+    let (alice_addr_32, alice_addr_mv) = addrs_from_ss58(ALICE_ADDR).unwrap();
 
-#[test]
-#[ignore = "to be implemented"]
-/// Test execution of a script with correct parameters but as a wrong user.
-/// Transaction does not require sudo but call was signed with sudo.
-fn execute_script_wrong_user() {
     new_test_ext().execute_with(|| {
-        assert_eq!(1, 0);
+        // Bob publishes the move-module 'Counter', test preparation.
+        let module = assets::read_module_from_project("get-resource", "Counter");
+        assert_ok!(MoveModule::publish_module(
+            RuntimeOrigin::signed(bob_addr_32.clone()),
+            module,
+            MAX_GAS_AMOUNT,
+        ));
+
+        // Check that there are no counters available.
+        assert_eq!(
+            get_vm_resource(&bob_addr_mv, "Counter", "Counter", &alice_addr_32).unwrap(),
+            None
+        );
```

Comment:
Here, but it can be applied in many below assertions as well
```suggestion
        assert!(get_vm_resource(&bob_addr_mv, "Counter", "Counter", &alice_addr_32).unwrap().is_none());
```

Reply:
Done